flake.nix

```diff
@@ -4,33 +4,41 @@
   inputs = {
     nixpkgs.url = github:NixOS/nixpkgs/nixos-23.11;
     flake-utils.url = github:numtide/flake-utils;
-    monoidal-functors.url = github:solomon-b/monoidal-functors/infra/flake-overlay-update;
+    monoidal-functors = {
+      url = github:solomon-b/monoidal-functors/solomon/removes-overlay;
+      flake = false;
+    };
```

Comment:
This is a flake but im using it with `callCabal2nix. I like the idea of fetching it as an input rather then using `fetchFromGitHub`.